app/controllers/api/v1/users_controller.rb

```
@@ -6,6 +6,25 @@ class UsersController < BaseController
         render json: current_user, status: :ok
       end
+
+      def update
+        if current_user.update(user_params)
+          render json: {
+            status: { code: 200, message: 'Usuário atualizado com sucesso.' },
+            data: UserSerializer.new(current_user).serializable_hash[:data][:attributes]
+          }, status: :ok
+        else
+          render json: {
+            status: { message: "Não foi possível atualizar o usuário. #{current_user.errors.full_messages.to_sentence}" }
```

RuboCop (1.73)

[convention] Line is too long. [121/120]

(Layout/LineLength)